auth/feature-tests/functional/step-definitions/attestation.steps.ts

```diff
@@ -0,0 +1,161 @@
+import 'dotenv/config'
+import { loadFeature, describeFeature } from "@amiceli/vitest-cucumber";
+import { AuthDriver } from "../driver/auth.driver";
+import { AuthFixtures } from "../fixtures/auth.fixtures";
+import { LoggingDriver } from '../driver/logging.driver';
+import { expect } from "vitest"
+import { AttestationDriver } from '../driver/attestation.driver';
+
+const feature = await loadFeature(
+  "feature-tests/functional/features/attestation.feature"
+);
+
+const mappedExamples: {
+  [key: string]: any;
+} = {
+  'valid': AuthFixtures.valid,
+  'invalid': AuthFixtures.invalid,
+  'missing': AuthFixtures.missing,
+  'expired': AuthFixtures.expired,
+  'unknown_app': AuthFixtures.unknown,
+}
+
+type ScenarioVariables = { [key: string]: any };
+
+const givenAnAppInitiatesLogin = (variables: ScenarioVariables, context: any) => {
+  context.user = mappedExamples[variables["user"]]
+}
+
+const whenARequestIsMadeWithAttestationHeader = async (
+  {
+    variables,
+    context,
+    authDriver,
+    attestationDriver,
+  }: {
+    variables: ScenarioVariables,
+    context: any,
+    authDriver: AuthDriver
+    attestationDriver: AttestationDriver
+  }
+) => {
+  if (variables["header"] === "valid") {
+    // generate fresh attestation token for valid use-cases
+    const { token } = await attestationDriver.getToken(process.env.FIREBASE_IOS_APP_ID!)
+    context.user['attestationToken'] = token
+  } else if (variables["header"] === "unknown") {
+    const { token } = await attestationDriver.getToken(process.env.UNKNOWN_ANDROID_APP_ID!)
```

Comment:
App ID has to be an app in our Firebase instance, but no configured in configuration repository.